src/qibo/backends/__init__.py

```diff
@@ -96,6 +95,13 @@ def get_backend():
         return "_".join([K.name, K.gates])
 
 
+if _BACKEND_NAME != "tensorflow": # pragma: no cover
+    # CI uses tensorflow as default backend
+    log.warning("Numpy does not support Qibo custom operators and GPU. "
```

Comment:
I think we can replace Numpy with the backend name. Ideally we should store the "allowed" support for each backend, so then this message may be more useful.